src/rest-server/src/routes/v2/job.js

```diff
@@ -39,6 +39,10 @@ router.route('/:frameworkName')
   /** GET /api/v2/jobs/:frameworkName - Get job */
   .get(controller.get);
 
+router.route('/:jobName/executionType')
+  /** PUT /api/v2/jobs/:frameworkName/executionType - Start or stop job */
+  .put(token.check, controller.execute);
```

Comment:
patch instead of put

Reply:
this is rest server's api, used `PATCH` in k8s apiserver's request